docs/en/ingest-management/release-notes/release-notes-8.12.asciidoc

```diff
@@ -22,6 +23,31 @@ Also see:
 * {kibana-ref}/release-notes.html[{kib} release notes]
 * {beats-ref}/release-notes.html[{beats} release notes]
 
+// begin 8.12.2 relnotes
+
+[[release-notes-8.12.2]]
+== {fleet} and {agent} 8.12.2
+
+Review important information about {fleet-server} and {agent} for the 8.12.2 release.
+
+[discrete]
+[[bug-fixes-8.12.2]]
+=== Bug fixes
+
+{fleet}::
```

Comment:
@juliaElastic looks like this is missing the release note for agents not being able to upgrade in Fleet server